### DISCUSSION TITLE
Patch Files...

### DISCUSSION THREAD

**Discussion by Kreytan** [humhue/DAOTools#13] [Q&A] [answered]
Hi thank you very much for the tool, no matter the years there will always be DAO players. 

One question, how can I patch single files or whole folders?

**Comment by humhue** [comment, marked as answer]
I'd suggest you to create an .erf file with the DAToolset and then patch that. You can then extract single files from the erf using the toolset.
Another way would be to change the code slightly, or simply running the specific file patcher based on the type of file you want to patch, as shown here: https://github.com/humhue/DAOTools/blob/991f8c76f5fb1dca573bfbe37e3fcfa230fc85d3/gff/dlg_patcher.py#L75
You could run the command "python3 path/to/dlg_patcher.py path/to/dir/with/dlg/files" and it should patch all dlg files in that dir.

I was working on a GUI for the patcher, but it's so radically different from the current codebase that I would need a lot of testing before releasing it. I'm currently very busy.
I hope these other methods work for you.
